torch/csrc/jit/passes/quantization/insert_observers.cpp

```diff
@@ -718,8 +750,10 @@ graph(%self, %a, %b):
           inplace_add_nn_relu,   inplace_add_f_relu,
           add_aten_relu,         add_aten_relu_,
           inplace_add_aten_relu, inplace_add_aten_relu_,
-          nn_bn_nn_relu,         nn_bn_f_relu,
-          nn_bn_aten_relu,       nn_bn_aten_relu_,
+          nn_bn2d_nn_relu,       nn_bn2d_f_relu,
```

Comment:
are we adding bn1d later?

Reply:
if this is used by user then we should implement them. right now we don't have plans to do that

Reply:
right now quantized::batch_norm1d is not implemented